solidity_adapter/src/index/mod.rs

```diff
@@ -289,11 +293,12 @@ impl FSEntity {
     ///
     fn delete(&self, current: &Path) -> anyhow::Result<()> {
         if let Self::Directory(_) = self {
-            fs::remove_dir_all(current)
-                .map_err(|err| anyhow::anyhow!("Failed to delete directory: {}", err))?;
+            fs::remove_dir_all(current).map_err(|err| {
+                anyhow::anyhow!("Failed to delete directory ({:?}): {}", current, err)
```

Comment:
Maybe remove the `()`. `{:?}` for paths adds its own double quotes.

Reply:
Yes, that would look better (I was trying to be unintrusive.)  Here’s the current output:

> Failed to update tests: Failed to delete directory ("/Users/flash/scr2/abiEncoderV1"): No such file or directory (os error 2)

Reply:
But I deleted some of the changes when cleaning up my debug output, 31ce8cd9, which I’ll correct.

Reply:
f4b65d9